scheduler/test/cook/test/testutil.clj

```diff
@@ -28,14 +28,21 @@
 (defn run-test-server-in-thread
   "Runs a minimal cook scheduler server for testing inside a thread. Note that it is not properly kerberized."
   [conn port]
+  (cook.components/init-logger {:file "log/test-server.log"
```

Comment:
Is there a more central place we could put this? In other words, instead of just adding logging support to this spot, can we add it for any/all unit tests?